lang/jp/chapters/enabling_extensions.adoc

```diff
@@ -0,0 +1,101 @@
+// Copyright 2019-2022 The Khronos Group, Inc.
+// SPDX-License-Identifier: CC-BY-4.0
+
+ifndef::chapters[:chapters:]
+
+[[enabling-extensions]]
+= 拡張機能の有効化
+
+ここでは、拡張機能の有効化について説明します。
+
+== 2種類の拡張機能
+
+拡張機能には、**インスタンス拡張機能**と**デバイス拡張機能**の2つのグループがあります。簡単に言えば、**インスタンス拡張機能** は `VkInstance` 全体に結び付けられ、**デバイス拡張機能** は単一の `VkDevice` インスタンスにのみ結び付けられます。
+
+この情報は、各拡張機能のリファレンスページの「`Extension Type`」セクションに記載されています。以下に例を示します。
+
+image::../../../chapters/images/enabling_extensions_instance_extension.png[enabling_extensions_instance_extension.png]
+
+== 対応の確認
+
+アプリケーションは、まずlink:https://www.khronos.org/registry/vulkan/specs/1.3-extensions/html/vkspec.html#extendingvulkan-extensions[物理デバイスにクエリ]し、`vkEnumerateInstanceExtensionProperties` または `vkEnumerateDeviceExtensionProperties` で拡張機能が**サポート**されているかどうかを確認することができます。
```

Comment:
もし品題で`対応`を使ったら、どうしてここで`対応`のかわりに`サポート`を使った？

@gpx1000 maybe this brings up a good future logistical question of if someone else (who's Japanese is better than mine) has a question about the translation who is able to respond to such questions

Reply:
You're asking the right questions...

This is likely to be a problem, however, I seem to remember @marty-johnson59 mentioning that Khronos might have someone on staff.  I think the process will be we bring up all issues/questions to the samples working group and from there we'll route to hopefully someone who can help.

Reply:
If you used correspondence in your article, why did you use support instead of correspondence here? <-- From google translate this is the question.  I wonder if logistically it would make sense to run through Google translate the questions / replies and have a native speaker double check?

Reply:
So without going deep into Japanese, they translated "support" as カタカナ and 漢字 and not sure which is normally used more often, but surprised it wasn't consistent in both spots...also Google translate is an amazing tool, but not for everything such as technical documentation

Reply:
I agree, which is really the reasoning for having a request for support from a native speaker to handle the translations.
For the broader point of how to support questions dynamically, best idea I have is to provide best efforts via bringing it back to the WG, looking for resources within Khronos and member companies to find someone capable of addressing the issues as they come up.  I am wholly unqualified to even pretend to translate Japanese let alone any other language so I'd be unable to really do much more than that.

Reply:
ah, in case of support, using サポート or 対応 is not a big issue for understanding the context since both does mean the same thing in Japanese. So, it's not a big issue, and no Japanese would care about the difference between サポート and 対応 in a sentence.
Could you please accept as is rather than taking more time to discuss here? We can modify/update if there is serious translation mismatch. Thanks!

Reply:
I agree, rather have something tangible out and can always adjust, (cc @marty-johnson59 )